FHEM/14_SD_WS_Maverick.pm

```diff
@@ -240,14 +211,12 @@ SD_WS_Maverick_Parse($$)
   # Den SensorState bei Inaktivität zurücksetzen lassen durch Timer 
   my $inactivityinterval=int(AttrVal($name,"inactivityinterval",360));
   if ($sensor_1_state ne "unknown") {
-    $hash->{sensor_1_state}=$sensor_1_state;
-    RemoveInternalTimer($hash, 'SD_WS_Maverick_SetSensor1Inaktiv');
-    InternalTimer(time()+($inactivityinterval), 'SD_WS_Maverick_SetSensor1Inaktiv', $hash, 0);
+    RemoveInternalTimer($hash, 'SD_WS_Maverick_SetSensor1Inactive');
+    InternalTimer(time()+($inactivityinterval), 'SD_WS_Maverick_SetSensor1Inactive', $hash, 0);
   }
   if ( $sensor_2_state ne "unknown") {
-    $hash->{sensor_2_state}=$sensor_2_state;
-    RemoveInternalTimer($hash, 'SD_WS_Maverick_SetSensor2Inaktiv');
-    InternalTimer(time()+($inactivityinterval), 'SD_WS_Maverick_SetSensor2Inaktiv', $hash, 0);
+    RemoveInternalTimer($hash, 'SD_WS_Maverick_SetSensor2Inactive');
```

Comment:
Könntest Du 
RemoveInternalTimer ... noch in die _Undef sub mit aufnehmen? Das fehlt leider noch vom Vorgänger.

Reply:
Ist erledigt.